[2024-12-09T20:50:10.808+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T20:50:10.828+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.load_data manual__2024-12-09T20:49:55.370737+00:00 [queued]>
[2024-12-09T20:50:10.838+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.load_data manual__2024-12-09T20:49:55.370737+00:00 [queued]>
[2024-12-09T20:50:10.838+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-09T20:50:10.852+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-09 20:49:55.370737+00:00
[2024-12-09T20:50:10.862+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=960) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-09T20:50:10.864+0000] {standard_task_runner.py:72} INFO - Started process 962 to run task
[2024-12-09T20:50:10.864+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb_complex', 'load_data', 'manual__2024-12-09T20:49:55.370737+00:00', '--job-id', '331', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb_complex.py', '--cfg-path', '/tmp/tmpypa4n10v']
[2024-12-09T20:50:10.867+0000] {standard_task_runner.py:105} INFO - Job 331: Subtask load_data
[2024-12-09T20:50:11.235+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb_complex.load_data manual__2024-12-09T20:49:55.370737+00:00 [running]> on host 8553af9ff67b
[2024-12-09T20:50:11.324+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb_complex' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T20:49:55.370737+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-09T20:49:55.370737+00:00'
[2024-12-09T20:50:11.325+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-09T20:50:11.326+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-09T20:50:11.327+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-12-09 20:50:10.829724+00:00
[2024-12-09T20:50:11.327+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex and current dag run status:running
[2024-12-09T20:50:11.328+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T20:50:11.395+0000] {logging_mixin.py:190} INFO - Données chargées dans MongoDB.
[2024-12-09T20:50:11.396+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-09T20:50:11.407+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-09T20:50:11.408+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb_complex, task_id=load_data, run_id=manual__2024-12-09T20:49:55.370737+00:00, execution_date=20241209T204955, start_date=20241209T205010, end_date=20241209T205011
[2024-12-09T20:50:11.418+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-09T20:50:11.419+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-09T20:50:11.419+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex queued_at:2024-12-09 20:49:55.378942+00:00
[2024-12-09T20:50:11.420+0000] {logging_mixin.py:190} INFO - Task hostname:8553af9ff67b operator:PythonOperator
[2024-12-09T20:50:11.445+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-09T20:50:11.647+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-09T20:50:11.650+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
